Tests/XcodeGenKitTests/BreakpointGeneratorTests.swift

```diff
@@ -0,0 +1,19 @@
+import ProjectSpec
+import Spectre
+import XCTest
+
+class BreakpointGeneratorTests: XCTestCase {
+
+    func testBreakpoints() {
+        describe {
+
+            $0.it("generates breakpoint") {
+                let breakpoint = Breakpoint(type: .exception)
+                let project = Project(basePath: "", name: "test", targets: [], breakpoints: [breakpoint])
+                let xcodeProject = try project.generateXcodeProject()
+                guard let xcbreakpoint = xcodeProject.sharedData?.breakpoints?.breakpoints.first else { throw failure("Breakpoint not found") }
```

Comment:
The current codebase has since moved to using `unwrap`
```swift
let xcbreakpoint = try unwrap(xcodeProject.sharedData?.breakpoints?.breakpoints.first)
```

Reply:
@yonaskolb 
I will fix this issue at the end to avoid resolving conflicts multiple times.
Is it fine to merge `master` to here?
Or should I rebase `master` to here?

Reply:
Merging master instead of rebasing is good, so the changes are visible